quick_start.adoc

```diff
@@ -329,9 +329,9 @@ An entry under `identityProviders` has the following fields:
   * `clientID` contains the client ID of a client registered with the OpenID provider.
   * `clientSecret` contains a reference to a secret object containing the OpenID client secret. 
   - `claims` maps attributes on the user entry.
-    * `preferredUsername` is the list of attributes whose values should be used as the preferred username. Optional. If unspecified, the preferred username is determined from the value of the sub claim
-    * `name` is the list of attributes whose values should be used as the display name. Optional. If unspecified, no display name is set for the identity
-    * `email` is the list of attributes whose values should be used as the email address. Optional. If unspecified, no email is set for the identity
+    * `preferredUsername` is the list of attributes whose values should be used as the preferred username. Optional. If unspecified, the preferred username is determined from the value of the sub claim. Openshift support array of preferredUsername, but we are only support first value from the array.
```

Comment:
Please tweak the 3 new lines:
OpenShift supports an array of <preferredUsername/name/email>, but identity configuration management for Kubernetes is only supporting the first value from the array.